tests/test_python_grammar.py

```diff
@@ -0,0 +1,221 @@
+from unittest import TestCase, main
+
+from lark import Lark
+from lark.indenter import PythonIndenter
+from lark.exceptions import UnexpectedCharacters, UnexpectedToken
+
+
+python_parser = Lark.open_from_package(
```

Comment:
I think we should delay this till the tests actually get called. When we at some point break Lark's init, we would probably prefer if the dedicated tests fail and not the import itself.

Reply:
Good point. It should be in the class `__init__`.

Reply:
`__init__`? It should probably be `setUpClass`.

Reply:
definitely `setUpClass`